src/ActionModal/components/ActionModal.vue

```diff
@@ -651,6 +651,15 @@ export default {
     },
     async submit() {
       this.submissionError = null
+
+      if (
```

Comment:
different idea I want to share: How about returning `{error: true}` to avoid this fancy checking if the object is empty? :D